<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AssociatedPlans</class>
 <widget class="QDialog" name="AssociatedPlans">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>486</width>
    <height>242</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Associated Plans</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="CtlListWidget" name="planList"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="CtlButton" name="btnOk">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Ok</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="CtlButton" name="btnClose">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
  <customwidgets>
    <customwidget>
      <class>CtlButton</class>
      <extends>QPushButton</extends>
      <header>bus/common_control/src/button/CtlButton.h</header>
    </customwidget>CtlListWidget
    <customwidget>
      <class>CtlListWidget</class>
      <extends>QListWidget</extends>
      <header>bus/common_control/src/itemview/CtlListWidget.h</header>
    </customwidget>
  </customwidgets>
 <resources/>
 <connections/>
</ui>
